contracts/reputationMiningCycle/ReputationMiningCycleCommon.sol

```diff
@@ -142,19 +143,20 @@ contract ReputationMiningCycleCommon is ReputationMiningCycleStorage, PatriciaTr
   }
 
   uint256 constant UINT256_MAX = 2**256 - 1;
-  uint256 constant SUBMITTER_ONLY_WINDOW_DURATION = 60 * 10;
-  uint256 constant Y = UINT256_MAX / SUBMITTER_ONLY_WINDOW_DURATION;
+  uint256 constant CHALLENGE_RESPONSE_WINDOW_DURATION = 60 * 20;
```

Comment:
Am I correct in thinking that this change doubles the amount of time a dispute will take? Is that acceptable in terms of colony performance (e.g. not having reputation be frozen for a long time during a dispute)?

Reply:
Is there some middle ground here, like 6 + 6?

Reply:
Security, not speed, is the priority here. If I had my way, all these windows would be a lot longer than they are.